sherpa-onnx/csrc/sherpa-onnx.cc

```diff
@@ -61,29 +69,28 @@ for a list of pre-trained models to download.
 
   sherpa_onnx::OnlineRecognizer recognizer(config);
 
-  float duration = 0;
+  std::vector<Stream> ss;
+  // std::vector<sherpa_onnx::OnlineStream *> p_ss;
```

Comment:
Could you remove the commented code?

Reply:
@csukuangfj  Just removed the commented code. Thanks.